From 1141563abafa369fec7db018a7c8c00e219990cc Mon Sep 17 00:00:00 2001
From: Peter Yin <peteryin.openbmc@gmail.com>
Date: Fri, 14 Jun 2024 13:36:36 +0800
Subject: [PATCH 1005/1010] ARM: dts: aspeed: Harma: read cpu temp and power

Add sbrmi and sbtsi device

There are no sbrmi and sbtsi drivers in upstream,
so it only patch in Meta tree.

Signed-off-by: Peter Yin <peteryin.openbmc@gmail.com>
---
 arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-harma.dts | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-harma.dts b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-harma.dts
index 65421d18299c..32e4d4530e9a 100644
--- a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-harma.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-harma.dts
@@ -376,6 +376,16 @@ imux23: i2c@1 {
 
 &i2c7 {
 	status = "okay";
+	sbrmi@3C {
+		compatible = "amd,sbrmi";
+		reg = <0x3C>;
+		dimm-count = <8>;
+		dimm-ids = <0x00 0x01 0x02 0x03 0x10 0x11 0x12 0x13>;
+	};
+	sbtsi@4C {
+		compatible = "amd,sbtsi";
+		reg = <0x4C>;
+	};
 };
 
 &i2c8 {
-- 
2.44.2

